styles/footer.module.scss

```diff
@@ -0,0 +1,13 @@
+.footer {
+  border-top: 1px solid $border-grey;
+  padding: 20px 0;
```

Comment:
nie mozna uzuwac tych paddingow bo pozniej nam sie nie beda szerokosci zgadzały, border lepiej dac na before i wlasciwoscia width: calc(100% -30px)

Reply:
ale to. jest padding na wysokosc

Reply:
ale wyrzuciłeś padding z row 15px

Reply:
sorki z container